pipeline/inferrer/Dockerfile

```diff
@@ -1,36 +1,73 @@
-FROM public.ecr.aws/docker/library/python:3.10-slim as base
+ARG pythonversion=2.7
```

Comment:
Should this be 3.10?

Reply:
Actually, I'll remove it.  There's a story as to why I set it to 2.7, but I think it's not relevant now after some other changes.